packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -137,7 +150,12 @@ export default function Sidebar({
         {
           icon: <ListIcon Icon={SettingsIcon} />,
           title: 'Customize',
-          action: () => setShowSettings(!showSettings),
+          action: () => {
+            trackEvent({
```

Comment:
Maybe it's better to just unify the way we track modals. And add the relevant event in the settings component rather than here. What do you think?
We can also introduce a custom hook for tracking the state of a modal

Reply:
Agreed, might be better to do it inside the modals as this will be better for wherever they might get opened from.